SQL Server Deployment Fails

**Describe the bug**
When deploying the data domain with the ADO pipeline, the "Deploy SQL Server 001" step fails with the following error:

```
##[error]PECsNotExistingToDenyPublicNetworkAccess: Unable to set Deny Public Network Access to Yes since there is no private endpoint enabled to access the server. Please set up private endpoints and retry the operation (https://docs.microsoft.com/azure/sql-database/sql-database-private-endpoint-overview#how-to-set-up-private-link-for-azure-sql-database).
```

--- a/README.md
+++ b/README.md
@@ -235,7 +235,7 @@ The following table explains each of the parameters:
 | **DATA_LANDING_ZONE_SUBSCRIPTION_ID**        | Specifies the subscription ID of the Data Landing Zone where all the resources will be deployed | `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` |
 | **DATA_LANDING_ZONE_NAME**                   | Specifies the name of your Data Landing Zone. The value should consist of alphanumeric characters (A-Z, a-z, 0-9) and should not contain any special characters like `-`, `_`, `.`, etc. Special characters will be removed in the renaming process. | `mynode01` |
 | **LOCATION**                                 | Specifies the region where you want the resources to be deployed. | `northeurope` |
-| **SUBNET_ID**                              | Specifies the resource ID of the dedicated privatelink-subnet which was created during the Data Landing Zone deployment. The subnet should be configured with `privateEndpointNetworkPolicies` and `privateLinkServiceNetworkPolicies`, as mentioned in the *Prerequisites*| `/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/my-network-rg/providers/Microsoft.Network/virtualNetworks/my-vnet/subnets/{my}-privatelink-subnet` |
+| **SUBNET_ID**                                | Specifies the resource ID of the dedicated privatelink-subnet which was created during the Data Landing Zone deployment. The subnet should be configured with `privateEndpointNetworkPolicies` and `privateLinkServiceNetworkPolicies`, as mentioned in the *Prerequisites* | `/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/my-network-rg/providers/Microsoft.Network/virtualNetworks/my-vnet/subnets/{my}-privatelink-subnet` |
 |**SYNAPSE_STORAGE_ACCOUNT_NAME**| Specifies the name of the Azure Synapse Storage Account | `synapsestorageaccount`
 |**SYNAPSE_STORAGE_ACCOUNT_FILE_SYSTEM_NAME**| Specifies the name of the Synapse Account filesystem| `fs`| 
 | **AZURE_RESOURCE_MANAGER_CONNECTION_NAME**   | Specifies the resource manager connection name in Azure DevOps. You can leave the default value if you want to use GitHub Actions for your deployment. More details on how to create the resource manager connection in Azure DevOps can be found in step 4. b) or [here](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/connect-to-azure?view=azure-devops#create-an-azure-resource-manager-service-connection-with-an-existing-service-principal). | `my-connection-name` |
@@ -317,12 +317,12 @@ If you are using Azure DevOps Pipelines, you can navigate to the pipeline that y
 
 | File/folder                   | Description                                |
 | ----------------------------- | ------------------------------------------ |
-| `.ado/workflows`              | Folder for ADO workflows. The `dataDomainDeployment.yml` workflow shows the steps for an end-to-end deployment|
-| `.github/workflows`           | Folder for GitHub workflows. The `updateParameters.yml` sample workflow shows you how you can update the parameters with your preffered naming, while the `dataDomainDeployment.yml` workflow shows the steps for an end-to-end deployment |
-| `code`                        | Sample Password Generation code that will be run in the deployment workflow of the resources that needs a Password during the creation  |
-| `configs`                     | Folder containing the `config.json` file containing details of all the resources files which will be deployed in this repository. This file will be reference by the `UpdateParameters.ps1` script in order to update the paramenters name before the deployment |
+| `.ado/workflows`              | Folder for ADO workflows. The `dataDomainDeployment.yml` workflow shows the steps for an end-to-end deployment of the architecture. |
+| `.github/workflows`           | Folder for GitHub workflows. The `updateParameters.yml` workflow is used for the parameter update process, while the `dataDomainDeployment.yml` workflow shows the steps for an end-to-end deployment of the architecture. |
+| `code`                        | Sample password generation script that will be run in the deployment workflow for resources that require a password during the deployment. |
+| `configs`                     | Folder containing a script and configuration file that is used for the parameter update process. |
 | `docs`                        | Resources for this README.                 |
-| `infra`                       | Folder containing zll the ARM templates for each of the resources that will be deployed (`deploy.{resource}.json`) together with their parameter files (`params.{resource}.json`). 
+| `infra`                       | Folder containing all the ARM templates for each of the resources that will be deployed (`deploy.{resource}.json`) together with their parameter files (`params.{resource}.json`). |
 | `CODE_OF_CONDUCT.md`          | Microsoft Open Source Code of Conduct.     |
 | `LICENSE`                     | The license for the sample.                |
 | `README.md`                   | This README file.                          |
@@ -372,7 +372,7 @@ This error message appears, in case during the deployment it tries to create a t
 
 **Solution:**
 
-This error message appears during the deployment of a resource, in case the subnet associated with that specific resource has not been configured as per the **Prerequisites** or if the `SUBNET_ID` parameter from <a href="/.github/workflows/updateParameters.yml">`/.github/workflows/updateParameters.yml`</a>, which specifies the resource ID of the dedicated subnet is not the right one. 
+This error message appears during the deployment of a resource, in case the subnet associated with that specific resource has not been configured as per the **Prerequisites**. Please update the subnet configuration or set the correct `SUBNET_ID` parameter in <a href="/.github/workflows/updateParameters.yml">`/.github/workflows/updateParameters.yml`</a>. 
 
 
 # Contributing

